src/leiningen/i18n.clj

```diff
@@ -53,10 +53,10 @@
   (let [dest (path-join (dev-resources-dir project) "Makefile.i18n")
         makefile (io/resource "leiningen/i18n/Makefile")]
     (spit (io/as-file dest)
-          (clojure.string/replace
+          (clojure.string/replace-first
            (slurp makefile)
-           #"PACKAGE=.*"
-           (str "PACKAGE=" (bundle-package-name project))))))
+           #"(?m)^BUNDLE=.*$"                               ; (?m) = set the multiline flag for the RegExp
```

Comment:
The comment is a little hard to understand. Would be better to say something like 'use ?m so $ matches end of line rather than end of string'